azext_iot/_params.py

```diff
@@ -762,6 +762,16 @@ def load_arguments(self, _):
             options_list=["--iot-hubs", "--ih"],
             help="Host name of target IoT Hub. Use space-separated list for multiple IoT Hubs.",
         )
+        context.argument(
+            "webhook_url",
+            options_list=["--webhook-url", "--wh"],
+            help="The webhook URL used for custom allocation requests.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            help="The API version of the provisioning service in the custom allocation request.",
```

Comment:
I think we should add a tiny bit more detail for this argument. Specifically mentioning 'types' and the minimum supported version.

Something like:

"The API version of the provisioning service **types** sent in the custom allocation request. Minimum supported version: 2018-09-01-preview"

Reply:
Updated